wpilibc/src/main/native/cpp/ADXL345_I2C.cpp

```diff
@@ -92,15 +92,12 @@ ADXL345_I2C::AllAxes ADXL345_I2C::GetAccelerations() {
   return data;
 }
 
-void ADXL345_I2C::InitSendable(nt::NTSendableBuilder& builder) {
+void ADXL345_I2C::InitSendable(wpi::SendableBuilder& builder) {
   builder.SetSmartDashboardType("3AxisAccelerometer");
-  builder.SetUpdateTable(
-      [this, x = nt::DoubleTopic{builder.GetTopic("X")}.Publish(),
-       y = nt::DoubleTopic{builder.GetTopic("Y")}.Publish(),
-       z = nt::DoubleTopic{builder.GetTopic("Z")}.Publish()]() mutable {
-        auto data = GetAccelerations();
-        x.Set(data.XAxis);
-        y.Set(data.YAxis);
-        z.Set(data.ZAxis);
-      });
+  builder.AddDoubleProperty(
+      "X", [this]() mutable { return GetAccelerations().XAxis; }, nullptr);
```

Comment:
Unfortunately this is not equivalent to the previous code, as GetAccelerations() will be called 3x, which will result in reading the device 3x as often.  I think this is true for all of the accelerometer/gyro classes.  Fixing this the right way will requires structured data support (#5391 plus Sendable support for it).

Reply:
What if I call GetAccelerations and store the data in a local variable? That’s basically the same as before, the difference is the variable scope.

Reply:
There's no place for the local variable to live; the property callbacks are all called individually.  Again, the better approach here will be to wait for the structured data support to be merged so it's just a single property.

Reply:
So then I’ll take out those bits and keep everything else, which will be the CommandScheduler, the ADIS16448_IMU, and the ADIS16470_IMU.